### PR TITLE
Feature/hamburger nav

### PR DESCRIPTION
**Pull request by mkazmer**
hamburger nav is functioning and basically styled

### CODE REVIEW

File: client/src/components/Header.scss [hunk 1]
```diff
@@ -1,36 +1,114 @@
...
+  }
 
+  .menu-wrap {
+    display: flex;
+    position: fixed;
+    padding: 0;
+    right: 0%;
+    top: 0%;
+    z-index: 1;
+    .toggler {
```
Comment (acousineau) [inline client/src/components/Header.scss]: As a rule of thumb - just a formatting thing - be sure to stick an empty line in between each new defintion. Makes it easier to read

File: client/src/components/Header.scss [hunk 2]
```diff
@@ -1,36 +1,114 @@
...
+      position: absolute;
+      right: 0%;
+      top: 10px;
+      z-index: 2;
+      cursor: pointer;
+      width: 50px;
+      height: 50px;
+      opacity: 0;
+      &:checked ~ .main-menu {
```
Comment (acousineau) [inline client/src/components/Header.scss]: Same here

File: client/src/components/Header.scss [hunk 3]
```diff
@@ -1,36 +1,114 @@
...
+      right: 0;
+      transition: all 0.4s ease;
+      }
+      &:hover:checked + .hamburger > div {
```
Comment (acousineau) [inline client/src/components/Header.scss]: Here..

File: client/src/components/Header.scss [hunk 4]
```diff
@@ -1,36 +1,114 @@
...
+          background-color: darken(white, 30%)
+      }
+      &:checked + .hamburger > div {
```
Comment (acousineau) [inline client/src/components/Header.scss]: Here... etc... you get the idea

**Comment by mkazmer** [comment]
made the 'X' turn into SABERS!

**Pull request merged**